tests/acceptance/api_test.go

```diff
@@ -1092,8 +1092,6 @@ func TestApi(t *testing.T) {
 					"metadata": map[string]interface{}{
 						"foo": "bar",
 					},
-					// this shouldn't actually persist
-					"accountStatus": "hippos",
```

Comment:
accountStatus isn't settable via update.  Previously this would just skip and stlil write the rest of the data.  Going forward this will cause a validation error.

Reply:
we should be careful about breaking API changes....
Can we at least make sure this is called out in the CHANGELOG, that we're breaking the API interface in the next version?